packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -0,0 +1,544 @@
+   */
+  readonly minCapacity?: AuroraCapacityUnit;
+
+  /**
+   * The maximum capacity for an Aurora serverless database cluster.
+   *
+   * @default - determined by Aurora based on database engine
+   */
+  readonly maxCapacity?: AuroraCapacityUnit;
+
+  /**
+   * The time before an Aurora serverless database cluster is paused.
+   * A database cluster can be paused only when it is idle (it has no connections).
+   *
+   * If a DB cluster is paused for more than seven days, the DB cluster might be
+   * backed up with a snapshot. In this case, the DB cluster is restored when there
+   * is a request to connect to it.
+   *
+   * Set to 0 to disable
+   *
+   * @default - automatic pause enabled after 5 minutes
+   */
+  readonly autoPause?: Duration;
+}
+
+/**
+ * New or imported Serverless Cluster
+ */
+abstract class ServerlessClusterBase extends Resource implements IServerlessCluster {
+  /**
+   * Identifier of the cluster
+   */
+  public abstract readonly clusterIdentifier: string;
+
+  /**
+   * The endpoint to use for read/write operations
+   */
+  public abstract readonly clusterEndpoint: Endpoint;
+
+  /**
+   * The endpoint to use for read/write operations
+   */
+  public abstract readonly clusterReadEndpoint: Endpoint;
+
+  /**
+   * Access to the network connections
+   */
+  public abstract readonly connections: ec2.Connections;
+
+  /**
+   * Renders the secret attachment target specifications.
+   */
+  public asSecretAttachmentTarget(): secretsmanager.SecretAttachmentTargetProps {
+    return {
+      targetId: this.clusterIdentifier,
+      targetType: secretsmanager.AttachmentTargetType.RDS_DB_CLUSTER,
+    };
+  }
+}
+
+/**
+ * Create an Aurora Serverless Cluster
+ *
+ * @resource AWS::RDS::DBCluster
+ */
+export class ServerlessCluster extends ServerlessClusterBase {
+
+  /**
+   * Import an existing DatabaseCluster from properties
+   */
+  public static fromServerlessClusterAttributes(scope: Construct, id: string,
+    attrs: ServerlessClusterAttributes): IServerlessCluster {
+
+    return new ImportedServerlessCluster(scope, id, attrs);
+  }
+
+  public readonly clusterIdentifier: string;
+  public readonly clusterEndpoint: Endpoint;
+  public readonly clusterReadEndpoint: Endpoint;
+  public readonly connections: ec2.Connections;
+
+  /**
+   * The secret attached to this cluster
+   */
+  public readonly secret?: secretsmanager.ISecret;
+
+  private readonly securityGroups: ec2.ISecurityGroup[];
+  private readonly subnetGroup: CfnDBSubnetGroup;
+  private readonly vpc: ec2.IVpc;
+  private readonly vpcSubnets?: ec2.SubnetSelection;
+
+  private readonly singleUserRotationApplication: secretsmanager.SecretRotationApplication;
+  private readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
+
+  constructor(scope:Construct, id: string, props: ServerlessClusterProps) {
+    super(scope, id);
+
+    this.vpc = props.vpc ?? new ec2.Vpc(this, 'Vpc', { maxAzs: 2 });
```

Comment:
`vpc` looks to be a required property.
```suggestion
    this.vpc = props.vpc;
```

Reply:
good catch